bundles/org.openhab.core.persistence/src/main/java/org/eclipse/smarthome/core/persistence/internal/PersistenceManagerImpl.java

```diff
@@ -283,11 +283,12 @@ private void initialize(Item item) {
                                         genericItem.removeStateChangeListener(this);
                                         genericItem.setState(historicItem.getState());
                                         genericItem.addStateChangeListener(this);
-                                        logger.debug("Restored item state from '{}' for item '{}' -> '{}'",
-                                                new Object[] {
-                                                        DateFormat.getDateTimeInstance()
-                                                                .format(historicItem.getTimestamp()),
-                                                        item.getName(), historicItem.getState().toString() });
+                                        if (logger.isDebugEnabled()) {
```

Comment:
:+1:

I thought about suggesting this check too because I don't know how "expensive" the called methods are.